### ISSUE TITLE
[Feature request] "Pseudo-pinned" mode for low-priority notifications

### ISSUE THREAD

**Issue by stuartpb** [heruoxin/Clip-Stack#29]
Going together with #27 and #28, it would be ideal if Clip Stack, when showing low-priority, iconless, bottom-of-the-list notifications, would allow for the ability to dismiss them, and have them *reappear* the next time the notification list is opened, the way Google Now notifications work.

**Comment by heruoxin** [comment]
Notification will reappear when system clipboard or clip 'stack' changed.

**Comment by stuartpb** [comment]
Yes, that's the point of this feature request, to make it so the notification will *always* reappear, while still allowing the notification to be dismissed (like Google Now's notifications, and to allow for special behavior on dismissal like #28).

**Comment by stuartpb** [comment]
I guess #27, #28, and #29 would best go together by replacing the "Notification" and "Pin notification" toggles with two multi-option selections (and a toggle for #27): "Notification" becoming a selection of "Show on change", "Never show", or "Always show", and "Pin notification" becoming "Dismissing notification", with options "Clear notification", "Delete clip", and "Not dismissable".

**Comment by heruoxin** [comment]
Your suggestion sounds really great, but Android system only allow you to dismiss the hole notification, not a part of it. It may be not well when user dismiss the hole notification (including up to 5 clips) but only delete the first clip.
![image](https://cloud.githubusercontent.com/assets/1999794/7396713/e8905400-eed5-11e4-90ad-9ceac62249ec.png)
(Sorry I'm Chinese so my English may be not well...:) )

**Comment by stuartpb** [comment]
I see - I was thinking of a model where Clip Stack only shows whatever is currently on the system clipboard (the top item) in the notification.

**Comment by stuartpb** [comment]
I'll probably write another feature request later for an option to limit the number of items that appear on the notification (I'd limit it to 1 on my phone).

Is it possible to put more than 1 button next to a part of a notification (next to the copy/share buttons)?
